dependencies.gradle

```diff
@@ -1,6 +1,6 @@
 ext {
     supportVersion = '25.3.1'
-    buildToolsVersionVariable = "25.0.3"
+    buildToolsVersionVariable = "27.0.0"
```

Comment:
You can remove this line, and all three occurrences of `buildToolsVersion` from the module's `build.gradle` files. Android Gradle plugin `3.0.0` will automatically pick the right version.

Reply:
@friederbluemle good to know :-)

Reply:
Oh, I haven't notice this too. Thank you.

https://developer.android.com/studio/releases/build-tools.html

> If you're using Android plugin for Gradle 3.0.0 or higher, your project automatically uses a default version of the build tools that the plugin specifies.

I remove this.

Reply:
When deleting buildToolsVersion, 26.0.2 was used.
I think that it is better to use the latest version if possible. So I specify 27.0.0.

Reply:
Interesting. I'm all for using the latest version if possible, but to be honest I think since Google implemented (and recommends to use) this auto-chooser, they know best which version to use for a particular Android Gradle plugin and compile/target SDK version combination.